IFC 4.3/ProjectSetup_1/UT_ProjectSetup_1.ifc

```diff
@@ -5,7 +5,7 @@ ISO-10303-21;
 /* ------- Stefan Jaud, stefan.jaud@tum.de, 2020.07.15. ---------------------------------------- */
 
 HEADER;
-FILE_DESCRIPTION (('ViewDefinition [NotDefinedYet]'), '2;1');
```

Comment:
ViewDefinition should not have been removed. Just the list should be empty. ViewDefinition[] is mandatory.

Reply:
Resolved 1152f70